gencred/merge_kubeconfig_secret.py

```diff
@@ -114,12 +151,18 @@ def validateArgs(args):
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='Merges the provided kubeconfig file into a kubeconfig file living in a kubernetes secret in order to add new cluster contexts to the secret. Requires kubectl and base64.') #pylint: disable=line-too-long
+    parser = argparse.ArgumentParser(
+        description='Merges the provided kubeconfig file into a kubeconfig file living in a kubernetes secret in order to add new cluster contexts to the secret. Requires kubectl and base64.')  # pylint: disable=line-too-long
```

Comment:
This should be updated to note the new google-cloud-secret-manager requirement.

Reply:
This is good to know, updated